worlds/animal_well/options.py

```diff
@@ -132,14 +132,49 @@ class WheelTricks(Choice):
     default = 0
 
 
-class WeirdTricks(Toggle):
+class BallThrowing(Choice):
     """
-    Include performing "weird" tricks in the logic.
-    Some of these tricks are difficult, tedious, or inconsistent.
-    Use at your own risk.
+    Include in logic using the ball to hit switches or buttons not ""designed"" for it.
+    Off means the ball will rarely be used for anything other than breaking blocks, spikes or guard shields.
+    Simple means the ball can be used to hit easy targets without any real rebound, setup or moving. Most ""yoyo chute"" buttons are included here.
+    Advanced means hitting your target may require bouncing off a wall, or moving while throwing to adjust momentum or get a good angle.
+    Expert includes any more complicated tricks, including those that require specific setups or getting lucky.
+    """									
+    internal_name = "ball_throwing"
+    display_name = "Ball Throwing"
+    option_off = 0
+    option_simple = 1
+    option_advanced = 2
+    option_expert = 3
+    default = 0
```

Comment:
```suggestion
    default = 1
```
I'm going to assume most players are totally fine with easy

Reply:
Agreed. Think this might have been something that got lost in the recovery.